### PR TITLE
benchmark: add aita, cache type copy

### PR DESCRIPTION
**Pull request by pared**
Benchmark for adding the dataset to cache with copy type:
https://dvc.org/blog/a-public-reddit-dataset

### CODE REVIEW

**Comment by efiop** [comment]
@pared Is there a way to make GHA render the results, btw?

**Comment by pared** [comment]
@efiop Ill research that!

File: benchmarks/benchmarks.py [hunk 1]
```diff
@@ -23,10 +27,41 @@ def time_init(self):
...
 
-        git = Repo.init(self.test_directory.name)
+        git = GitRepo.init(self.test_directory.name)
         git.close()
 
     def time_init(self):
         main(["init"])
+
+
+class AITABench(BaseBench):
```
Comment (efiop) [inline benchmarks/benchmarks.py:38]: Maybe let's create separate files for different benchmarks? Should be easier to manage. Or does that not work with asv?

Comment (pared) [reply benchmarks/benchmarks.py:38]: Should be fine AFAIK

**Comment by pared** [comment]
Closing for now, will focus on randomly-generated data for now.

**Pull request closed without merging**